charts/shoot-core/components/charts/kube-proxy/templates/kube-proxy-cleanup-script.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kube-proxy-cleanup-script
+  namespace: kube-system
+  labels:
+    app: kubernetes
+    gardener.cloud/role: system-component
+    origin: gardener
+    role: proxy
+data:
+  cleanup.sh: |
+    #!/bin/sh -e
+    OLD_KUBE_PROXY_MODE=$(cat "$1")
```

Comment:
```suggestion
    OLD_KUBE_PROXY_MODE="$(cat "$1")"
```

Reply:
ping, this is not resolved yet

Reply:
I forgot this, thx for checking. Is now resolved.